dogm/demo/simulator/mapping/opengl/renderer.cpp

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: MIT
 // See accompanying LICENSE file for detailed information
 
-#include "dogm/opengl/renderer.h"
+#include "mapping/opengl/renderer.h"
 
 #define _USE_MATH_DEFINES
```

Comment:
I'm getting `warning: "_USE_MATH_DEFINES" redefined` during compilation. This is coming up now because you moved the simulator to the demo target, where this variable is already set in Cmake. Are you able to compile without this line?